services/feeds.js

```diff
@@ -0,0 +1,160 @@
+var request = require('superagent');
+
+/**
+ * @constructor
+ */
+
+function Feeds(defaults) {
+
+	defaults = defaults || {};
+
+	/* set default format or lang for all requests */
+
+	/* we default to json since it's a js-sdk */
+	this.format = defaults.format || 'json';
+	this.lang = defaults.lang;
+}
+
+/**
+ * Creates an object with the option feed params
+ * for this request.
+ * @returns {Object}
+ */
+
+Feeds.prototype.params = function (args) {
+	var params;
+
+	args = args || {};
+
+	params = {
+		format: args.format || this.format
+	};
+
+	/* don't include unless explicitly set */
+	if (args.lang || this.lang) {
+		params.lang = args.lang || this.lang;
+	}
+	return params;
+};
+
+/**
+ * Asserts that `key` is present in `args`.
+ * @param {Object} args
+ * @param {String} key
+ * @throws {Error}
+ */
+
+function assert(args, key) {
+	if (!args || !args.hasOwnProperty(key)) {
+		throw new Error('Missing required argument "' + key + '"');
+	}
+}
```

Comment:
I like the consistency here, but it would be nice to DRY this up by making this a lib module to share with REST.